src/server/memory_cmd.cc

```diff
@@ -389,6 +392,27 @@ void MemoryCmd::Track(CmdArgList args) {
     return;
   }
 
+  if (sub_cmd == "ADDRESS") {
+    string_view ptr_str = parser.Next();
+    if (parser.HasError()) {
+      return cntx_->SendError(parser.Error()->MakeReply());
+    }
+
+    size_t ptr = 0;
+    if (!absl::SimpleHexAtoi(ptr_str, &ptr)) {
+      return cntx_->SendError("Address must be hex number");
+    }
+
+    atomic_bool found;
```

Comment:
An atomics default constructor doesn't initialize it, it should be `found {false}`;

Reply:
I did NOT know that! Nice catch, thanks!

Reply:
We joked about C++ this morning 😄